packages/react-router/lib/dom/ssr/routes.tsx

```diff
@@ -473,89 +473,95 @@ export function createClientRoutes(
         };
       }
 
-      if (route.hasClientMiddleware) {
-        dataRoute.unstable_lazyMiddleware = async () => {
-          invariant(route);
-          let clientMiddlewareModule = await import(
-            /* @vite-ignore */
-            /* webpackIgnore: true */
-            route.clientMiddlewareModule || route.module
-          );
-          invariant(
-            clientMiddlewareModule?.unstable_clientMiddleware,
-            "No `unstable_clientMiddleware` export in chunk"
+      let lazyRoutePromise:
```

Comment:
This allows us to dedupe calls to `getLazyRoute` across the different lazy property loaders. This is only required because we have additional logic beyond calling `await import('...')`.